app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumNewPostActivityTest.kt

```diff
@@ -55,7 +60,41 @@ class ForumNewPostActivityTest {
         }.join() // Adding a timeout is not possible on Kotlin for the same silly reasons
     }
 
+    @Test
+    fun notifiedWhenPostReplyWorks() {
```

Comment:
Makes a little more sense I think
```suggestion
    fun notifiedOnPostReplyWorks() {
```